docs/src/pages/[platform]/getting-started/troubleshooting/shared/vite-polyfill-ts.mdx

```diff
@@ -0,0 +1,15 @@
+**3.** (Optional) If you run into TypeScript errors importing `aws-exports.js`, you may need to update the `tsconfig.json` file with the following config and add a type declaration file:
+
+```json
+  "compilerOptions": {
+    "allowJs": true,
+  }
+...
```

Comment:
Nit: what do you think about 
```suggestion
```
so that we have don't have invalid json syntax?